PluginDirectories/1/fpm.bundle/README.markdown

```diff
@@ -0,0 +1,24 @@
+#Flashlight Package Manager
+
+A simple decentralized package manager for Flashlight.
+
+Flashlight Package Manager allows Flashlight users to install any plugin that is hosted on Github. In order to work with Flashlight Package Manager the plugin must be in its own repository and the root of the repository must contain the `info.json` file.
+
+It is good practice to add a `version` property to `info.json`. It is not used at the moment, it's just displayed in Spotlight, however versioning will be supported in the future.
+
+## Supported commands
+
+### Install plugins
+
+    fpm install ~id(a)
```

Comment:
I see that ~id can be any github path ($user/$project) — would it be possible to have the command accept full github URLs too? it'd be nice to be able to copy+paste from the browser.